extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php

```diff
@@ -123,6 +125,15 @@ public function getWikiDomain( $sCurrentUrl ) {
 	}
 
 	/**
+	 * Returns a core of a full language code (e.g. pt from pt-br)
+	 * @param  string $sFullLangCode Full language code
+	 * @return string                A core of the language code
+	 */
+	public function getLanguageCore( $sFullLangCode ) {
+		return array_values( explode( '-', $sFullLangCode ) )[0];
```

Comment:
Good question. I think that I had this solution stuck in my head as non-working. Will simplify, thanks!